docs/current/162770-faq.md

```diff
@@ -111,14 +111,14 @@ Follow these steps:
   brew uninstall dagger
   ```
 
-1. Remove the `dagger` sub-directory of your `$XDG_CACHE_HOME` and `$XDG_CONFIG_HOME` directories. These are Dagger cache locations containing the auto-provisioned Dagger CLI (used for SDK operations).
+1. Remove the `dagger` sub-directory of your `$XDG_CACHE_HOME` and `$XDG_CONFIG_HOME` directories.
```

Comment:
Sorry, I should clarify this but leave it to you if you want to change or not.

The [XDG env vars](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) are used in Linux, but not the others. While some people may set it in macOS (maybe for portability), it's not something you can expect to find. It's used so that linux users can override the defaults for various paths in their system, by exporting a few specifically named environment variables. Usually programs that are installed in the system search first for these variables and then default to whatever is the default in each XDG variable (by convention).

To understand a bit more, see how the python library sets the paths for each platform:

### Unix
Notice that it checks first for `$XDG_CACHE_HOME` then uses the default of `~/.cache`:
```python
    def user_cache_dir(self) -> str:
        """
        :return: cache directory tied to the user, e.g. ``~/.cache/$appname/$version`` or
         ``~/$XDG_CACHE_HOME/$appname/$version``
        """
        path = os.environ.get("XDG_CACHE_HOME", "")
        if not path.strip():
            path = os.path.expanduser("~/.cache")  # noqa: PTH111
        return self._append_app_name_and_version(path)
```

https://github.com/platformdirs/platformdirs/blob/7035de4344d2ae4e55774c4bdec838f0a59a03fd/src/platformdirs/unix.py#L90-L98

### macOS

Doesn't use `$XDG_CACHE_HOME`, just `~/Library/Caches`:

```python
    def user_cache_dir(self) -> str:
        """:return: cache directory tied to the user, e.g. ``~/Library/Caches/$appname/$version``"""
        return self._append_app_name_and_version(os.path.expanduser("~/Library/Caches"))
```

https://github.com/platformdirs/platformdirs/blob/7035de4344d2ae4e55774c4bdec838f0a59a03fd/src/platformdirs/macos.py#L44-L46

### Windows

That depends on some windows env vars:

```python
    def user_cache_dir(self) -> str:
        """
        :return: cache directory tied to the user (if opinionated with ``Cache`` folder within ``$appname``) e.g.
         ``%USERPROFILE%\\AppData\\Local\\$appauthor\\$appname\\Cache\\$version``
        """
        path = os.path.normpath(get_win_folder("CSIDL_LOCAL_APPDATA"))
        return self._append_parts(path, opinion_value="Cache")
```


https://github.com/platformdirs/platformdirs/blob/7035de4344d2ae4e55774c4bdec838f0a59a03fd/src/platformdirs/windows.py#L72-L78

Reply:
You could add something like "or the equivalent for your platform".

Reply:
Added